### PR TITLE
Patch graviAURA strategy to HiddenHand's V2 Changes

### PR DESCRIPTION
**Pull request by sajanrajdev**
Relates to https://github.com/Badger-Finance/badger-multisig/issues/1335

## Context
Hidden Hands recently introduced new versions of their contracts that [broke](https://github.com/Badger-Finance/badger-multisig/issues/1335#issuecomment-1629713843) graviAURA's claiming flow. The following changes were made to handle the broken changes:
- HH doesn't handle native tokens anymore (any ETH bribes will be handled as wETH from the get go and tagged as such). The reference of the "BRIBES_VAULT" was removed from the claim function along with the separation of ETH special case - no longer applicable.
- Claiming tests checking this special case handling were generalized.
- The upgrade test was adapted to perform a claim of the latest rewards on fork (real claiming data from last round was included) - **Test is set to skip: The skipping [line](https://github.com/Badger-Finance/vested-aura/blob/994f8439ca2c38b789662761cd8eba6d19c00182/tests/upgrade/test_upgrade.py#L49) must be manually commented out to run!**  
- Linting fixes were applied to the affected python files and all of the Solidity files.

## Test results
All tests are passing:
image

### CODE REVIEW

File: tests/integration/test_claim_bribes.py [hunk 1]
```diff
@@ -244,14 +255,28 @@ def test_claim_bribes_with_redirection(
     # Confirm GNO event
     assert event[1]["destination"] == gno_recepient
     assert event[1]["token"] == gno
-    assert event[1]["amount"] == gno_recepient_balance_before + gno_amount - expected_gno_fee_amount
+    assert (
+        event[1]["amount"]
+        == gno_recepient_balance_before + gno_amount - expected_gno_fee_amount
```
Comment (rayeaster) [inline tests/integration/test_claim_bribes.py]: a typo `recipient` ?

File: tests/upgrade/test_upgrade.py [hunk 2]
```diff
@@ -1,34 +1,55 @@
...
 
-def test_check_storage_integrity(strat_proxy, vault_proxy, deployer, proxy_admin, proxy_admin_gov):
+
+@pytest.fixture
+def strategist(strat_proxy):
+    return accounts.at(strat_proxy.strategist(), force=True)
+
+
+@pytest.fixture
+def usdc():
```
Comment (rayeaster) [inline tests/upgrade/test_upgrade.py:44]: is this function `usdc()` locally needed?

File: tests/upgrade/test_upgrade.py [hunk 3]
```diff
@@ -46,15 +67,52 @@ def test_check_storage_integrity(strat_proxy, vault_proxy, deployer, proxy_admin
...
+    ]
 
-    ## Let's do a quick earn and harvest as well
-    vault_proxy.earn({"from": gov})
-    strat_proxy.harvest({"from": gov})
\ No newline at end of file
+    strat_proxy.claimBribesFromHiddenHand(
+        NEW_REWARDS_DISTRIBUTOR, metadata, {"from": strategist}
+    )
+    # Confirm that claimable amount was claimed and transferred to the Bribes Processor
```
Comment (rayeaster) [inline tests/upgrade/test_upgrade.py:115]: does it make sense to generalize a local function like `def confirmBribeClaim()` to wrap following lines and allow to specify the target token to be confirmed?

**Review by rayeaster — APPROVED** [review]
LGTM to remove special handling of native ETH processing and add format linting for both solidity and test files

File: tests/upgrade/test_upgrade.py (hunk 2 above)
Comment (sajanrajdev) [reply tests/upgrade/test_upgrade.py:44]: It is used at a few points during the upgrade test to ensure that the bribes are properly harvested and transferred to the processor.

File: tests/upgrade/test_upgrade.py (hunk 3 above)
Comment (sajanrajdev) [reply tests/upgrade/test_upgrade.py:115]: I thought of making a generalized test but given that this should be the last time we upgrade (it is 2 months away from deprecation) I decided to go through the hardcoded route and just test the current situation (only USDC bribes to claim).

File: tests/integration/test_claim_bribes.py (hunk 1 above)
Comment (sajanrajdev) [reply tests/integration/test_claim_bribes.py]: Good catch fixing on the test but leaving the strat untouched to reduce the changes

**Comment by sajanrajdev** [comment]
@rayeaster, comments addressed and test with typo fix passing:
image

**Comment by sajanrajdev** [comment]
A workaround was found to process bribes without the patch via script adaptations: https://github.com/Badger-Finance/badger-multisig/pull/1349

**Pull request closed without merging**